.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
Determine the node version that should be used, because otherwise, Travis CI uses a version like `0.x.x` which is obviously way too old.

Reply:
How about adding the version in the `.travis.yml`?
This way multiple versions could be tested (e.g. 8, 10, 12)

```yml
language: node_js
node_js: 
  - 10
```

According to the [docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) using the `.nvmrc` is the optional way to specificy the node version.

When using travis I thing it's cleaner to have it in the travis-config. When using the node-version for other things (changing CI system, utilizing the node-version for...whatever) using `.nvmrc` might be a more agnostic way

Reply:
@fuhlig 

> When using the node-version for other things (changing CI system, utilizing the node-version for...whatever) using .nvmrc might be a more agnostic way

That’s why I choose to use a `.nvmrc` — to make sure that other potential build/ci tools in our chain are forced to use the current Node.js LTS version.

If we want to test different node versions on Travis CI in the future, we can still do that:

> The `.nvmrc` file is only read when `node_js` key in your `.travis.yml` files does not specify a nodejs version.
>
> — https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc

But to choose an agnostic default Node.js version for the project for the time being — independent from any tools and testing scenarios — seems more pragmatic to me.